From a74c3e1db4752426bc4bbc8bbc14f08dee6f086d Mon Sep 17 00:00:00 2001
From: Milan Crha <mcrha@redhat.com>
Date: Mon, 9 Mar 2015 15:20:28 +0100
Subject: [PATCH 0463/1800] EContactEditorFullName - Do not realize the dialog
 in init()

That shows a runtime warning on the console, that the taskbar cannot
be set on already realized windows.
---
 addressbook/gui/contact-editor/e-contact-editor-fullname.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/addressbook/gui/contact-editor/e-contact-editor-fullname.c b/addressbook/gui/contact-editor/e-contact-editor-fullname.c
index cec35aad47..27543124b8 100644
--- a/addressbook/gui/contact-editor/e-contact-editor-fullname.c
+++ b/addressbook/gui/contact-editor/e-contact-editor-fullname.c
@@ -214,7 +214,6 @@ e_contact_editor_fullname_init (EContactEditorFullname *e_contact_editor_fullnam
 	action_area = gtk_dialog_get_action_area (dialog);
 	content_area = gtk_dialog_get_content_area (dialog);
 
-	gtk_widget_realize (GTK_WIDGET (e_contact_editor_fullname));
 	gtk_container_set_border_width (GTK_CONTAINER (action_area), 12);
 	gtk_container_set_border_width (GTK_CONTAINER (content_area), 0);
 
-- 
2.15.0

